### DISCUSSION TITLE
Starting SMplayer with Sofware Scaling video filter enabled

### DISCUSSION THREAD

**Discussion by m-spitteler** [smplayer-dev/smplayer#59] [Q&A]
Hi!
I have been using SMPlayer for years but I'm new to this discussion forum, please advise me if I inadvertently break a forum netiquette rule.

I want to SMPlayer ALWAYS start with "Video->Filters->Software enabled". When this option is disabled, subtitles look blurry when playing old lower-resolution videos. With the option enabled, subtitles look very sharp regardless video resolution. I want to do this automatically to avoid having to select the options from menus every time I open a new file.

I know there must be a way to pass a parameter to SMplayer or MPlayer to do this automatically, but I don't know how to do that. I researched the web to no avail, but I suspect something to do with "-zoom=yes" or "lavfi [graph=scale=1360:-2]" or the like, but all my attempts to put in the .ini files or "Preferences->Advanced->MPlayer/mpv" filter options end up crashing MPlayer or having no effect.

Thank you in advance.

**Comment by smplayer-dev** [comment]
Go to Preferences -> Advanced. Type this in the field to run actions whenever a file is opened:
`upscaling true`

**Comment by m-spitteler** [comment]
Thank you, it did the trick. Also putting scale=1360:-2 in the video filters produces the same effect (My screen is an old 1360x768 LED panel recycled as monitor). I wonder why this option it is not activated by default.
